sphinxcontrib/confluencebuilder/builder.py

```diff
@@ -177,10 +163,14 @@ def prepare_writing(self, docnames):
 
             doctitle = first_section[idx].astext()
             if not doctitle:
+                if self.publish:
+                    ConfluenceLogger.warn("document will not be published "
```

Comment:
Should we just generate a title? I mean I'd be concerned people would use this in CI/CD and never see that warning

Reply:
It's a possibility. I would lean towards having a more blunt auto-naming schema if this is desired. For example, `notitle-<docname>` or `autogen-<docname>` to prevent any collisions with randomly generated document names (or just a docname) with other pages that possibly shouldn't be overridden.

Maybe it would be ideal to have a configuration option which could disable such a capability as well (i.e. by default, automatically generate a title if none is set unless explicitly configured to not do so).

If this is desired, it'll probably be good to start a new issue to capture this new capability outside this PR. 🤔

Reply:
Agreed. It can be a separate feature.